### ISSUE TITLE
is_pending_review flag is missing on reports

### ISSUE THREAD

**Issue by al63** [CAVaccineInventory/vial#179]
Please close this if this is my ignorance about how vial works or how django admin panel works:

I noticed while looking through https://vial-staging.calltheshots.us/admin/core/report/, that there does not appear to be a `is_pending_review` field. I know that we've spent quit a bit of time on scooby making sure that flag is properly set if we have a suspect report.

See this function here in the netlify API call: https://github.com/CAVaccineInventory/help.vaccinate/blob/main/netlify/functions/submitReport/index.js#L24

We mark is_pending_review to true for all with the "Trainee" role and 15% with the "Journeyman" role, and based on a `validateReport` function. The client already will set `is_pending_review` using the same `validateReport` logic, so thats covered. It's just a matter of where the trainee/journeyman logic is done: on the client or on vial.

so to sum up:

1. Vial should support is_pending_review for QA teams
2. We need to determine if vial is responsible or if the client is responsible for marking `is_pending_review` automatically based on Trainee and Journeyman roles. I can prepare a branch for the outcome of it being the client responsibility.
